tesseract_ros/tesseract_monitoring/src/environment_monitor.cpp

```diff
@@ -473,7 +473,7 @@ void EnvironmentMonitor::newTesseractStateCallback(const tesseract_msgs::Tessera
       res.request.revision = static_cast<unsigned long>(environment->getRevision());
       if (get_monitored_environment_changes_client_.call(res))
       {
-        if (!applyEnvironmentCommandsMessage(res.response.id, environment->getRevision(), res.response.commands))
+        if (!tesseract_rosutils::processMsg(*(tesseract_->getEnvironment()), res.response.commands))
```

Comment:
Not sure I fully follow the reasoning for switching what function is being used here, but should this also be applied to lines 830 and 832? And if so, is the applyEnvironmentCommandsMessage function still necessary to exist?

Reply:
Discussed offline, good to go